packages/zipkin/src/tracer/index.js

```diff
@@ -106,7 +106,7 @@ class Tracer {
       parentId: new Some(parentId.spanId),
       spanId: randomTraceId(),
       debug: parentId.isDebug(),
-      sampled: parentId.sampled,
+      sampled: new Some(parentId.sampled),
```

Comment:
This will fail as now there is a nested `Some` property which will fail the check.
https://github.com/openzipkin/zipkin-js/blob/c80e9c00784a86945e5940068a312c3e6baeb9c3/packages/zipkin/src/option.js#L73-L76